src/io/flutter/run/SdkFields.java

```diff
@@ -25,13 +25,15 @@
  */
 public class SdkFields {
   private @Nullable String filePath;
+  private @Nullable String programArgs;
   private @Nullable String workDir;
 
-  public SdkFields() {}
+  public SdkFields() {
+  }
 
   /**
    * Creates SDK fields from a Dart file containing a main method.
-   *
+   * <p>
    * <p>(Automatically chooses the working directory.)
```

Comment:
This is generated by the formatter.  As a general rule we trust it or change it's settings.